### DISCUSSION TITLE
Contributions not showing

### DISCUSSION THREAD

**Discussion by mshokoya** [community/community#22847] [New to GitHub]
<p>whenever I create I new repo on GitHub, the very first push shows up on my contributions graph and contributions activity section.<br>
every push after the initial push is not recorded on my contributions graph and contributions activity section.</p>
<p>I know I meet the requirements specified by GitHub show my contributions should be showing</p>
<p>does anyone know how to fix this ?</p>
<p>Thanks in advance</p>

**Comment by pixeldesu** [comment, marked as answer]
<p>Are you using an e-mail address to push that is added to your GitHub account?</p>
<p>If it isn’t, contributions can’t be mapped back to you.</p>
<p>You can check your local email address for commits using</p>
<pre><code class="lang-auto">git config --global user.email
</code></pre>
<p>And check your emails on GitHub on here: <a href="https://github.com/settings/emails" rel="nofollow noopener">https://github.com/settings/emails</a></p>
<p>If the email you use for commits is present there, contributions will show up properly in your graph, and your user will also be properly linked in commit history in projects you contribute to.</p>
<p>Also, the project creation is always visible on GitHub as that push is a GitHub action they can of course map back towards you.</p>

**Comment by mshokoya** [comment]
<p>stupid me. I factory reset my laptop periodically and I forgot I needed to do this</p>
<p>thanks</p>

**Comment by Nihiru** [comment]
<p>Hey <a class="mention" href="/pixeldesu">@pixeldesu</a> I’m still facing this issue. Can you help me out?</p>

**Comment by ali-2032** [comment]
This repository has been disabled.

Access to this repository has been disabled by GitHub Staff due to a violation of GitHub's terms of service. If you are the owner of the repository, you may reach out to GitHub Support for more information.
